_partials/ubuntu_sqlite.md

```diff
@@ -0,0 +1,18 @@
+## SQLite
+
+[SQLite](https://sqlite.org/index.html) is a database engine used to execute CRUD commands on databse files. Let's install it with `homebrew` .
```

Comment:
If we put SQLite before PostgreSQL, then we can move the database intro sentence here and adapt it a bit: 

```suggestion
In a few weeks, we'll talk about databases and SQL. [SQLite](https://sqlite.org/index.html) is a database engine used to execute CRUD commands on database files. Let's install it:
```

Then we could update the PostgreSQL section intro sentence with something like: 
> Sometimes, SQLite is not enough and you will need a more advanced too called PostgreSQL, an open-source, robust and production-ready database system.